### DISCUSSION TITLE
Inputs Category collapsed by default

### DISCUSSION THREAD

**Discussion by Profana92** [storybookjs/storybook#27124] [Ideas]
### Describe the solution you'd like

I would like to be able to collapse categories by default:

![Screenshot 2024-05-14 at 14 28 03](https://github.com/storybookjs/storybook/assets/87131514/b185b8bc-6fe8-4daa-b646-06559d808b3b)

### Are you able to assist to bring the feature to reality?

no

**Comment by joelisfar** [comment]
I support this as an option! In my case I think having the first one open by default could be nice with the others hidden. Or I could just not put items in a category if I want them always showing.
